src/main/java/yoda/Ui.java

```diff
@@ -0,0 +1,41 @@
+package yoda;
+
+import yoda.commands.Command;
+import yoda.exceptions.InvalidInputException;
+
+import java.util.Scanner;
+
+public class Ui {
```

Comment:
I'm thinking the `Ui` class should just be responsible for printing stuff and the actual logic of running the program, creating tasklist, scanner, and parser should be handled by another class? 🤔

Reply:
Thank you for your feedback! I have made the changes you suggested.